--- a/src/pcp2elasticsearch/GNUmakefile
+++ b/src/pcp2elasticsearch/GNUmakefile
@@ -28,7 +28,7 @@
 install: default
 ifeq "$(HAVE_PYTHON_REQUESTS)" "true"
 	$(INSTALL) -m 755 $(TARGET).py $(PCP_BIN_DIR)/$(TARGET)
-	$(INSTALL) -S $(BASHDIR)/pcp $(BASHDIR)/$(TARGET)
+	$(INSTALL) -S pmclient $(BASHDIR)/$(TARGET)
 	@$(INSTALL_MAN)
 endif

--- a/src/pmseries/GNUmakefile
+++ b/src/pmseries/GNUmakefile
@@ -34,7 +34,7 @@
 	$(INSTALL) -m 755 -d $(PCP_SYSCONF_DIR)/pmseries
 	# compatibility symlink (share pmproxy config Redis server list)
 	$(INSTALL) -S $(PCP_SYSCONF_DIR)/pmproxy/pmproxy.conf $(PCP_SYSCONF_DIR)/pmseries/pmseries.conf
-	$(INSTALL) -S $(BASHDIR)/pcp $(BASHDIR)/$(CMDTARGET)
+	$(INSTALL) -S pmclient $(BASHDIR)/$(CMDTARGET)
 else
 build-me:
 install:
--- a/src/pmdumptext/GNUmakefile
+++ b/src/pmdumptext/GNUmakefile
@@ -25,7 +25,7 @@
 else
 	$(INSTALL) -m 755 $(BINARY) $(PCP_BIN_DIR)/$(COMMAND)
 endif
-	$(INSTALL) -S $(BASHDIR)/pcp $(BASHDIR)/$(COMMAND)
+	$(INSTALL) -S pmclient $(BASHDIR)/$(COMMAND)
 else
 build-me:
 install:
--- a/src/bashrc/GNUmakefile
+++ b/src/bashrc/GNUmakefile
@@ -28,9 +28,9 @@
 install:	default
 	$(INSTALL) -d $(PCP_BASHSHARE_DIR)
 	$(INSTALL) -d $(BASHDIR)
-	$(INSTALL) -m 644 $(BASHRC) $(BASHDIR)/pcp
+	$(INSTALL) -m 644 $(BASHRC) $(BASHDIR)/pmclient
 	for c in `echo $(COMMANDS)`; do \
-	    $(INSTALL) -S $(BASHDIR)/pcp $(BASHDIR)/$$c; \
+	    $(INSTALL) -S pmclient $(BASHDIR)/$$c; \
 	done

 default_pcp:	default
--- a/src/pcp2graphite/GNUmakefile
+++ b/src/pcp2graphite/GNUmakefile
@@ -30,7 +30,7 @@
 install: default
 ifeq "$(HAVE_PYTHON)" "true"
 	$(INSTALL) -m 755 $(TARGET).py $(PCP_BIN_DIR)/$(TARGET)
-	$(INSTALL) -S $(BASHDIR)/pcp $(BASHDIR)/$(TARGET)
+	$(INSTALL) -S pmclient $(BASHDIR)/$(TARGET)
 	@$(INSTALL_MAN)
 endif

--- a/src/pcp2zabbix/GNUmakefile
+++ b/src/pcp2zabbix/GNUmakefile
@@ -30,7 +30,7 @@
 install: default
 ifeq "$(HAVE_PYTHON)" "true"
 	$(INSTALL) -m 755 $(TARGET).py $(PCP_BIN_DIR)/$(TARGET)
-	$(INSTALL) -S $(BASHDIR)/pcp $(BASHDIR)/$(TARGET)
+	$(INSTALL) -S pmclient $(BASHDIR)/$(TARGET)
 	@$(INSTALL_MAN)
 endif

--- a/src/pcp2xlsx/GNUmakefile
+++ b/src/pcp2xlsx/GNUmakefile
@@ -28,7 +28,7 @@
 install: default
 ifeq "$(HAVE_PYTHON_OPENPYXL)" "true"
 	$(INSTALL) -m 755 $(TARGET).py $(PCP_BIN_DIR)/$(TARGET)
-	$(INSTALL) -S $(BASHDIR)/pcp $(BASHDIR)/$(TARGET)
+	$(INSTALL) -S pmclient $(BASHDIR)/$(TARGET)
 	@$(INSTALL_MAN)
 endif

--- a/src/pmrep/GNUmakefile
+++ b/src/pmrep/GNUmakefile
@@ -33,7 +33,7 @@
 	$(SUBDIRS_MAKERULE)
 	$(INSTALL) -m 755 $(SCRIPT) $(PCP_BIN_DIR)/$(TARGET)
 	$(INSTALL) -S $(PCP_BIN_DIR)/$(TARGET) $(PCP_BIN_DIR)/$(LNKTARGET)
-	$(INSTALL) -S $(BASHDIR)/pcp $(BASHDIR)/$(TARGET)
+	$(INSTALL) -S pmclient $(BASHDIR)/$(TARGET)
 	@$(INSTALL_MAN)
 endif

--- a/src/pcp2spark/GNUmakefile
+++ b/src/pcp2spark/GNUmakefile
@@ -30,7 +30,7 @@
 install: default
 ifeq "$(HAVE_PYTHON)" "true"
 	$(INSTALL) -m 755 $(TARGET).py $(PCP_BIN_DIR)/$(TARGET)
-	$(INSTALL) -S $(BASHDIR)/pcp $(BASHDIR)/$(TARGET)
+	$(INSTALL) -S pmclient $(BASHDIR)/$(TARGET)
 	$(INSTALL) -m 755 -d $(EX_DIR)
 	$(INSTALL) -m 644 README.md $(EX_DIR)/README.md
 	$(INSTALL) -m 644 $(COLLECTOR).py $(EX_DIR)/$(COLLECTOR).python
--- a/src/pcp2json/GNUmakefile
+++ b/src/pcp2json/GNUmakefile
@@ -30,7 +30,7 @@
 install: default
 ifeq "$(HAVE_PYTHON)" "true"
 	$(INSTALL) -m 755 $(TARGET).py $(PCP_BIN_DIR)/$(TARGET)
-	$(INSTALL) -S $(BASHDIR)/pcp $(BASHDIR)/$(TARGET)
+	$(INSTALL) -S pmclient $(BASHDIR)/$(TARGET)
 	@$(INSTALL_MAN)
 endif

--- a/src/pcp2influxdb/GNUmakefile
+++ b/src/pcp2influxdb/GNUmakefile
@@ -30,7 +30,7 @@
 install: default
 ifeq "$(HAVE_PYTHON_INFLUXDB)" "true"
 	$(INSTALL) -m 755 $(TARGET).py $(PCP_BIN_DIR)/$(TARGET)
-	$(INSTALL) -S $(BASHDIR)/pcp $(BASHDIR)/$(TARGET)
+	$(INSTALL) -S pmclient $(BASHDIR)/$(TARGET)
 	@$(INSTALL_MAN)
 endif

--- a/src/pcp2xml/GNUmakefile
+++ b/src/pcp2xml/GNUmakefile
@@ -28,7 +28,7 @@
 install: default
 ifeq "$(HAVE_PYTHON)" "true"
 	$(INSTALL) -m 755 $(TARGET).py $(PCP_BIN_DIR)/$(TARGET)
-	$(INSTALL) -S $(BASHDIR)/pcp $(BASHDIR)/$(TARGET)
+	$(INSTALL) -S pmclient $(BASHDIR)/$(TARGET)
 	@$(INSTALL_MAN)
 endif

