--- a/src/ws/Makefile-ws.am
+++ b/src/ws/Makefile-ws.am
@@ -184,9 +184,6 @@
 nodist_systemdunit_DATA += $(patsubst %.in,%,$(systemd_units_in))
 dist_systemdunit_DATA += src/ws/system-cockpithttps.slice

-firewalldir = $(prefix)/lib/firewalld/services
-firewall_DATA = src/ws/cockpit.xml
-
 motddir = $(datadir)/$(PACKAGE)/motd/
 dist_motd_DATA = src/ws/inactive.motd
 dist_motd_SCRIPTS = src/ws/update-motd
@@ -235,7 +232,6 @@
 EXTRA_DIST += \
 	$(systemd_units_in) \
 	src/ws/cockpit-desktop.in \
-	$(firewall_DATA) \
 	$(tempconf_in) \
 	$(NULL)

