--- a/coolwsd.service
+++ b/coolwsd.service
@@ -4,7 +4,7 @@

 [Service]
 EnvironmentFile=-/etc/sysconfig/coolwsd
-ExecStart=/usr/bin/coolwsd --version --o:sys_template_path=/opt/cool/systemplate --o:child_root_path=/opt/cool/child-roots --o:file_server_root_path=/usr/share/coolwsd
+ExecStart=/usr/bin/coolwsd --config-file=/etc/coolwsd/coolwsd.xml
 KillSignal=SIGINT
 TimeoutStopSec=120
 User=cool
@@ -13,7 +13,7 @@
 LimitNOFILE=infinity:infinity

 ProtectSystem=strict
-ReadWritePaths=/opt/cool /var/log
+ReadWritePaths=/var/log /var/lib/coolwsd /usr/share/coolwsd

 ProtectHome=yes
 PrivateTmp=yes
