Index: build/cmake/init.cmake
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/build/cmake/init.cmake b/build/cmake/init.cmake
--- a/build/cmake/init.cmake	(revision 9c0a8be1dc32063d91ed1901fd5fcd54f4f955a1)
+++ b/build/cmake/init.cmake	(date 1638252250492)
@@ -155,11 +155,6 @@
     else()
         set(lib_suffix "lib")
     endif()
-
-    # Include generator expression to suppress default Debug/Release pair
-    set(wxPLATFORM_LIB_DIR "$<1:/>${wxCOMPILER_PREFIX}${wxARCH_SUFFIX}_${lib_suffix}")
-else()
-    set(wxPLATFORM_LIB_DIR)
 endif()

 if(wxBUILD_CUSTOM_SETUP_HEADER_PATH)
Index: CMakeLists.txt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/CMakeLists.txt b/CMakeLists.txt
--- a/CMakeLists.txt	(revision 9c0a8be1dc32063d91ed1901fd5fcd54f4f955a1)
+++ b/CMakeLists.txt	(date 1638216908921)
@@ -7,7 +7,9 @@
 # Licence:     wxWindows licence
 #############################################################################

-cmake_minimum_required(VERSION 2.8.12)
+cmake_minimum_required(VERSION 3.0.0)
+include(GNUInstallDirs)
+

 if(NOT CMAKE_CONFIGURATION_TYPES)
     get_property(HAVE_MULTI_CONFIG_GENERATOR GLOBAL PROPERTY GENERATOR_IS_MULTI_CONFIG)
@@ -43,7 +45,7 @@
 # Initialize variables for quick access to wx root dir in sub dirs
 set(wxSOURCE_DIR ${CMAKE_CURRENT_SOURCE_DIR})
 set(wxBINARY_DIR ${CMAKE_BINARY_DIR})
-set(wxOUTPUT_DIR ${wxBINARY_DIR}/lib)
+set(wxOUTPUT_DIR ${wxBINARY_DIR}/${CMAKE_INSTALL_LIBDIR})

 # parse the version number from wx/version.h and include in wxMAJOR_VERSION and wxMINOR_VERSION
 file(READ include/wx/version.h WX_VERSION_H_CONTENTS)
Index: build/cmake/utils/CMakeLists.txt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/build/cmake/utils/CMakeLists.txt b/build/cmake/utils/CMakeLists.txt
--- a/build/cmake/utils/CMakeLists.txt	(revision 9c0a8be1dc32063d91ed1901fd5fcd54f4f955a1)
+++ b/build/cmake/utils/CMakeLists.txt	(date 1638207460255)
@@ -19,6 +19,7 @@
     wx_exe_link_libraries(wxrc wxbase)

     set_target_properties(wxrc PROPERTIES FOLDER "Utilities")
+    set_target_properties(wxrc PROPERTIES SUFFIX "-${wxMAJOR_VERSION}.${wxMINOR_VERSION}${EXE_SUFFIX}" )

     wx_install(TARGETS wxrc
         RUNTIME DESTINATION "bin"
@@ -30,21 +31,6 @@
     else()
         set(EXE_SUFFIX ${CMAKE_EXECUTABLE_SUFFIX})
     endif()
-
-    if(NOT MSVC)
-        install(CODE "execute_process( \
-            COMMAND ${CMAKE_COMMAND} -E rename \
-            ${CMAKE_INSTALL_PREFIX}/bin/wxrc${EXE_SUFFIX} \
-            ${CMAKE_INSTALL_PREFIX}/bin/wxrc-${wxMAJOR_VERSION}.${wxMINOR_VERSION}${EXE_SUFFIX} \
-            )"
-        )
-        install(CODE "execute_process( \
-            COMMAND ${CMAKE_COMMAND} -E create_symlink \
-            ${CMAKE_INSTALL_PREFIX}/bin/wxrc-${wxMAJOR_VERSION}.${wxMINOR_VERSION}${EXE_SUFFIX} \
-            ${CMAKE_INSTALL_PREFIX}/bin/wxrc${EXE_SUFFIX} \
-            )"
-        )
-    endif()
 endif()

 # TODO: build targets for other utils
Index: build/cmake/config.cmake
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/build/cmake/config.cmake b/build/cmake/config.cmake
--- a/build/cmake/config.cmake	(revision 9c0a8be1dc32063d91ed1901fd5fcd54f4f955a1)
+++ b/build/cmake/config.cmake	(date 1638216959928)
@@ -58,7 +58,7 @@
         COMMAND
         ${CMAKE_COMMAND} -E create_symlink
         "lib/wx/config/inplace-${TOOLCHAIN_FULLNAME}"
-        "${CMAKE_CURRENT_BINARY_DIR}/wx-config"
+        "${CMAKE_CURRENT_BINARY_DIR}/wx-config-${wxMAJOR_VERSION}.${wxMINOR_VERSION}${EXE_SUFFIX}"
         )
 endfunction()

@@ -66,9 +66,9 @@

     set(prefix ${CMAKE_INSTALL_PREFIX})
     set(exec_prefix "\${prefix}")
-    set(includedir "\${prefix}/include")
-    set(libdir "\${exec_prefix}/lib")
-    set(bindir "\${exec_prefix}/bin")
+    set(includedir "\${prefix}/${CMAKE_INSTALL_INCLUDEDIR}")
+    set(libdir "\${exec_prefix}/${CMAKE_INSTALL_LIBDIR}")
+    set(bindir "\${exec_prefix}/${CMAKE_INSTALL_BINDIR}")

     find_program(EGREP egrep)
     mark_as_advanced(EGREP)
Index: build/cmake/functions.cmake
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/build/cmake/functions.cmake b/build/cmake/functions.cmake
--- a/build/cmake/functions.cmake	(revision 9c0a8be1dc32063d91ed1901fd5fcd54f4f955a1)
+++ b/build/cmake/functions.cmake	(date 1638253030266)
@@ -78,9 +78,9 @@
     cmake_parse_arguments(wxCOMMON_TARGET_PROPS "DEFAULT_WARNINGS" "" "" ${ARGN})

     set_target_properties(${target_name} PROPERTIES
-        LIBRARY_OUTPUT_DIRECTORY "${wxOUTPUT_DIR}${wxPLATFORM_LIB_DIR}"
-        ARCHIVE_OUTPUT_DIRECTORY "${wxOUTPUT_DIR}${wxPLATFORM_LIB_DIR}"
-        RUNTIME_OUTPUT_DIRECTORY "${wxOUTPUT_DIR}${wxPLATFORM_LIB_DIR}"
+        LIBRARY_OUTPUT_DIRECTORY "${wxOUTPUT_DIR}${CMAKE_INSTALL_LIBDIR}"
+        ARCHIVE_OUTPUT_DIRECTORY "${wxOUTPUT_DIR}${CMAKE_INSTALL_LIBDIR}"
+        RUNTIME_OUTPUT_DIRECTORY "${wxOUTPUT_DIR}${CMAKE_INSTALL_LIBDIR}"
         )

     if(wxBUILD_PIC)
@@ -372,9 +372,9 @@

         # Setup install
         wx_install(TARGETS ${name}
-            LIBRARY DESTINATION "lib${wxPLATFORM_LIB_DIR}"
-            ARCHIVE DESTINATION "lib${wxPLATFORM_LIB_DIR}"
-            RUNTIME DESTINATION "lib${wxPLATFORM_LIB_DIR}"
+            LIBRARY DESTINATION "${CMAKE_INSTALL_LIBDIR}"
+            ARCHIVE DESTINATION "${CMAKE_INSTALL_LIBDIR}"
+            RUNTIME DESTINATION "${CMAKE_INSTALL_LIBDIR}"
             BUNDLE DESTINATION Applications/wxWidgets
             )
     endif()
@@ -525,7 +525,7 @@

     wx_set_common_target_properties(${target_name} DEFAULT_WARNINGS)
     if(NOT wxBUILD_SHARED)
-        wx_install(TARGETS ${name} ARCHIVE DESTINATION "lib${wxPLATFORM_LIB_DIR}")
+        wx_install(TARGETS ${name} ARCHIVE DESTINATION "${CMAKE_INSTALL_LIBDIR}")
     endif()
 endfunction()

@@ -810,7 +810,7 @@

             list(APPEND cmds COMMAND ${CMAKE_COMMAND}
                 -E copy ${wxSOURCE_DIR}/${SUB_DIR}/${data_src}
-                ${wxOUTPUT_DIR}/${wxPLATFORM_LIB_DIR}/${data_dst})
+                ${wxOUTPUT_DIR}/${CMAKE_INSTALL_LIBDIR}/${data_dst})
         endforeach()
         add_custom_command(
             TARGET ${target_name} ${cmds}
@@ -846,13 +846,13 @@
         FOLDER ${APP_FOLDER}
         )
     set_target_properties(${target_name} PROPERTIES
-        VS_DEBUGGER_WORKING_DIRECTORY "${wxOUTPUT_DIR}/${wxPLATFORM_LIB_DIR}"
+        VS_DEBUGGER_WORKING_DIRECTORY "${wxOUTPUT_DIR}/${CMAKE_INSTALL_LIBDIR}"
         )

     if(group STREQUAL Tests)
         add_test(NAME ${target_name}
             COMMAND ${target_name}
-            WORKING_DIRECTORY "${wxOUTPUT_DIR}/${wxPLATFORM_LIB_DIR}")
+            WORKING_DIRECTORY "${wxOUTPUT_DIR}/${CMAKE_INSTALL_LIBDIR}")
     endif()
 endfunction()

Index: build/cmake/install.cmake
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/build/cmake/install.cmake b/build/cmake/install.cmake
--- a/build/cmake/install.cmake	(revision 9c0a8be1dc32063d91ed1901fd5fcd54f4f955a1)
+++ b/build/cmake/install.cmake	(date 1638247558252)
@@ -29,27 +29,21 @@
 if(MSVC)
     wx_install(
         DIRECTORY "${wxSETUP_HEADER_PATH}"
-        DESTINATION "lib${wxPLATFORM_LIB_DIR}")
+        DESTINATION "${CMAKE_INSTALL_LIBDIR}")
 else()
     wx_install(
         DIRECTORY "${wxSETUP_HEADER_PATH}"
-        DESTINATION "lib/wx/include")
+        DESTINATION "${CMAKE_INSTALL_LIBDIR}/wx/include")

     wx_install(
         FILES "${wxOUTPUT_DIR}/wx/config/${wxBUILD_FILE_ID}"
-        DESTINATION "lib/wx/config"
+        DESTINATION "${CMAKE_INSTALL_LIBDIR}/wx/config"
         PERMISSIONS OWNER_EXECUTE OWNER_WRITE OWNER_READ
                     GROUP_EXECUTE GROUP_READ
                     WORLD_EXECUTE WORLD_READ
         )

     install(DIRECTORY DESTINATION "bin")
-    install(CODE "execute_process( \
-        COMMAND ${CMAKE_COMMAND} -E create_symlink \
-        ${CMAKE_INSTALL_PREFIX}/lib/wx/config/${wxBUILD_FILE_ID} \
-        ${CMAKE_INSTALL_PREFIX}/bin/wx-config \
-        )"
-    )
 endif()

 # uninstall target
@@ -62,7 +56,6 @@
 if(NOT TARGET ${UNINST_NAME})
     # these files are not added to the install manifest
     set(WX_EXTRA_UNINSTALL_FILES
-        "${CMAKE_INSTALL_PREFIX}/bin/wx-config"
         "${CMAKE_INSTALL_PREFIX}/bin/wxrc-${wxMAJOR_VERSION}.${wxMINOR_VERSION}"
         )

