--- a/src/slic3r/GUI/Tab.cpp
+++ b/src/slic3r/GUI/Tab.cpp
@@ -22,6 +22,7 @@

 #include <wx/bmpcbox.h>
 #include <wx/bmpbuttn.h>
+#include <wx/collpane.h>
 #include <wx/treectrl.h>
 #include <wx/imaglist.h>
 #include <wx/settings.h>
@@ -30,9 +31,11 @@
 #include <boost/algorithm/string/split.hpp>
 #include <boost/algorithm/string/predicate.hpp>

-#include <boost/algorithm/string/replace.hpp>#include <boost/algorithm/string/trim.hpp>
+#include <boost/algorithm/string/replace.hpp>
+#include <boost/algorithm/string/trim.hpp>
 #include <boost/filesystem.hpp>
 #include <boost/filesystem/path.hpp>
+#include <boost/lexical_cast.hpp>
 #include "wxExtensions.hpp"
 #include "PresetComboBoxes.hpp"
 #include <wx/wupdlock.h>
@@ -1970,10 +1973,10 @@
                 auto sizer = new wxBoxSizer(wxVERTICAL);
                 wxCollapsiblePane* collpane = new wxCollapsiblePane(parent, wxID_ANY, _L("Help / Details:"));
                 // add the pane with a zero proportion value to the 'sz' sizer which contains it
-                sizer->Add(collpane, 0, wxGROW | wxALL, 5);
+                sizer->Add(new wxSizerItem(collpane, 0, (wxGROW | wxALL), 5, nullptr));
                 // now add a test label in the collapsible pane using a sizer to layout it:
                 wxWindow* win = collpane->GetPane();
-                collpane->Bind(wxEVT_COLLAPSIBLEPANE_CHANGED, [tab](wxEvent&) {
+                collpane->GetPane()->Bind(wxEVT_COLLAPSIBLEPANE_CHANGED, [tab](wxEvent&) {
                     tab->Layout();
                 });

